packages/zip-it-and-ship-it/tsconfig.json

```diff
@@ -3,7 +3,15 @@
   "compilerOptions": {
     "outDir": "dist" /* Specify an output folder for all emitted files. */,
     "esModuleInterop": true,
-    "strict": true
+    "noImplicitAny": true,
+    "strictFunctionTypes": true,
+    "strictPropertyInitialization": true,
+    "useUnknownInCatchVariables": false,
+    "exactOptionalPropertyTypes": false,
+    "noImplicitReturns": false,
+    "noUncheckedIndexedAccess": false,
+    "noImplicitOverride": true,
+    "noPropertyAccessFromIndexSignature": false
```

Comment:
The intent here was presumably to opt into strict mode, but the tsconfig files get merged... so `"strict": true` here wasn't really doing anything, as the specific flags were all still disabled from the base config.

Reply:
I reenabled what I could but had to leave a few disabled 😞